src/current/_includes/v24.3/ui/databases-list-page.md

```diff
@@ -0,0 +1,60 @@
+
+### Search and filter databases
+
+By default, the Databases List Page displays up to 10 databases on page 1. If more than 10 databases exist, use the page navigation at the bottom of the page to display the additional databases.
+
+To search for specific databases, use the search field above the list table:
+
+1. Enter a string in the search box.
+1. Press `Enter`.
+
+    The list of databases is filtered by the string.
+
+{% include_cached new-in.html version="v24.3" %} To filter databases based on the nodes on which the database tables are located, use the nodes multi-select dropdown above the list table:
+
+1. Click the dropdown arrow.
+1. Select one or more nodes. You may need to scroll down for nodes in different regions. You may also type in the beginning of the node name to narrow the list.
+
+    The list of databases is filtered by the nodes selected.
+
+{% if page.cloud == true  -%}
+NOTE: Nodes multi-select dropdown is not available on Standard or Basic clusters. 
+{% endif %}
+
+### Refresh data
+
+{% include_cached new-in.html version="v24.3" %} The `system.table_metadata` table caches the necessary metadata of a table that populates the following:
+
+- [Databases List Page](#databases-list-page)
+- [Database Tables Tab](#tables-list-tab) of the [Database Details Page](#database-details-page)
+- [Overview Tab](#overview-tab) of the [Table Details Page](#table-details-page)
+
+The [cluster settings]({% link {{ version_prefix }}/cluster-settings.md %}) that control the refresh behaviour for the cached data in the `system.table_metadata` table are:
+
+- `obs.tablemetadata.automatic_updates.enabled` (default: `false`) enables automatic updates of the table metadata cache `system.table_metadata`.
+- `obs.tablemetadata.data_valid_duration` (default: `20m0s`) is the duration for which the data in `system.table_metadata` is considered valid.
+
+On the [Databases List Page](#databases-list-page) and the [Database Tables Tab](#tables-list-tab) of the [Database Details Page](#database-details-page), above the table of information, the **Last refreshed** indicator displays how long ago the cached data was last refreshed automatically (per the cluster settings) or manually.
+
+The cached data can be refreshed in the following ways:
+
+- Automatically: Set the cluster setting `obs.tablemetadata.automatic_updates.enabled` to `true`. The frequency of refreshes is controlled by the cluster setting `obs.tablemetadata.data_valid_duration`. Automatic updates is disabled by default to avoid unnecessarily consuming resources. However, if you want databases and tables information immediately when you visit the corresponding page, then enable automatic updates.
+- Manually: When any user clicks the **Refresh data** icon button to the right of the **Last refreshed** indicator. If a refresh job has been triggered, the **Refresh data** icon button will be disabled and the hover text will display `Data is currently refreshing`.
```

Comment:
Just FYI that this text will change from `Data is currently refreshing` to show the job's start time and the progress https://github.com/cockroachdb/cockroach/pull/133188. Release note there comments on this.

Reply:
updated